shacl/BrickEntityShapeBase.ttl

```diff
@@ -54,8 +56,11 @@ bsh:EquipmentShape a sh:NodeShape ;
     ];
     sh:property [
         sh:path brick:isPartOf;
-        sh:class brick:Equipment;
-        sh:message "A piece of Equipment's parts should be always other Equipment."
+        sh:or (
+            [ sh:class brick:Equipment ]
+            [ sh:class brick:System ]
+        ) ;
+        sh:message "An piece of Equipment can only be part of another piece of Equipment or a System."
```

Comment:
I am pretty sure this needs to be phrased as two separate `sh:property` shapes:

```turtle
sh:or (
  sh:property [ ... ] 
  sh:property [ ... ]
)
```

due to SHACL's requirement that the arguments to `sh:or` are, themselves, valid shapes